website/docs/client/python.md

```diff
@@ -216,9 +216,9 @@ environment variables:
 
 ## Built-in Transport Types
 
-### HTTP
+### HTTP Transport
 
-Allows sending events to HTTP endpoint, using [requests](https://requests.readthedocs.io/).
+The HTTP transport provides synchronous, blocking event emission using [httpx](https://www.python-httpx.org/). This is the default transport implementation suitable for most use cases where immediate event delivery and error handling are preferred.
```

Comment:
Well, sync http transport still uses `requests`

Reply:
Removed that part, what we use should be just an implementation detail.